### PR TITLE
3.2 Release notes

### PR DESCRIPTION
**Pull request by poojaSprophecy**

### CODE REVIEW

File: docs/release_notes/sept2023.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- Prophecy Scala libs version: 7.1.14
+
+### Features {#FeaturesRelease315}
+
+#### Introducing Package Hub
+
+We are excited to introduce the Package Hub with Prophecy version 3.2. This feature allows platform leads to create and share standardized Pipeline components, including custom Gems, Subgraphs, Datasets, Pipelines, and more. For instance, you can now ensure consistent data encryption in line with your company's policies. Simply create a new Project, add the required custom encryption Gems and components, and publish it as a Package. Other teams can easily add this as a dependency and utilize the components from the Package.
+
+:::info
+Please note that all Prophecy-provided Gems now follow the same concept of Packages. With this release, your existing Projects will depend on Prophecy-provided packages for Gems. Your existing Pipelines and Projects should seamlessly work with this upgrade. If you encounter any issues, please don't hesitate to reach out to our support team.
```
[the remark this answers is not in the document]
Comment (poojaSprophecy) [reply docs/release_notes/sept2023.md:23]: No.  the custom gem code would be updated automatically to work with our new packages. later if they want to move their custom gems to packages , they can copy the gem code to a package and publish it.

**Review by atbida — APPROVED** [review]
my comments could be incorporated into the full doc page for that feature (eg connections). Approving.

**Review by jainabhinav — APPROVED** [review]

File: docs/release_notes/sept2023.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## 3.2 (September 14, 2023)
+
+- Prophecy Python libs version: 1.6.1
+- Prophecy Scala libs version: 7.1.14
+
+### Features {#FeaturesRelease315}
+
+#### Introducing Package Hub
+
+We are excited to introduce the Package Hub with Prophecy version 3.2. This feature allows platform leads to create and share standardized Pipeline components, including custom Gems, Subgraphs, Datasets, Pipelines, and more. For instance, you can now ensure consistent data encryption in line with your company's policies. Simply create a new Project, add the required custom encryption Gems and components, and publish it as a Package. Other teams can easily add this as a dependency and utilize the components from the Package.
```
[the remark this answers is not in the document]
Comment (poojaSprophecy) [reply docs/release_notes/sept2023.md:20]: I think let's keep all the details in the detailed doc. And link the doc here once we have it ready.

File: docs/release_notes/sept2023.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+:::
+
+Stay tuned for detailed documentation on this exciting feature.
+
+#### Knowledge Graph - Connectors
+
+When you navigate to the Environment tab in the Project browser, you can browse the `Catalog tables` from the Databricks Fabric to which you are connected.
+These `Catalog Tables` are automatically refreshed in the background using connectors in the Fabric.
+We'll refresh the tables according to the defined refresh frequency in your Connector settings.
+This ensures that users no longer need to wait and manually refresh the Environment Tab while creating or modifying Pipelines.
```
[the remark this answers is not in the document]
Comment (poojaSprophecy) [reply docs/release_notes/sept2023.md:33]: @meiprophecy  do we have the connections doc? We should link it here once we have it ready.

**Pull request merged**